### ISSUE TITLE
Testing - test load 19K sample bundles

### ISSUE THREAD

**Issue by briandoconnor** [HumanCellAtlas/data-bundle-examples#9]
In the last sprint we only were able to load 22 sample bundles.  We're waiting a blue box fix.  But after this fix is in place, deploy a new system and load all of Jim's samples.

**Comment by kozbo** [comment]
Is this a duplicate of #232?

**Comment by briandoconnor** [comment]
@kozbo this was created as a new ticket since the last ticket only loaded 22 samples.

**Comment by briandoconnor** [comment]
Need to check in with @sampierson, I think he's staged data in the new account, right?  But I don't think he's onboarded the 19K samples into a staging instance of the HCA storage system.

**Comment by briandoconnor** [comment]
FYI, should update the manifest to include stable UUIDs

**Comment by kozbo** [comment]
@sampierson are you blocked by [#234](https://github.com/HumanCellAtlas/data-store/issues/234)?

**Comment by sampierson** [comment]
This is implemented and works for bundle with small files.  However the DSS API has now been made asynchronous for larger files and returns 202s, after which you are expected to poll your file repeatedly until it appears. `storer.py` needs to accommodate this new behavior.  I'll add that functionality this week.

**Comment by sampierson** [comment]
The test was performed.  Some errors were produced which generated other ticket.
